python/cudf/cudf/utils/dateutils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+import calendar
+
+from cudf.utils.dtypes import is_scalar
+
+day_names = list(calendar.day_name)
+month_names = list(calendar.month_name)
+
+
+def wkday_idx_to_name(inds_in):
+    indices = range(len(day_names) + 1)
+    ind_to_day = dict(zip(indices, day_names))
+    if is_scalar(inds_in):
+        return ind_to_day[inds_in]
```

Comment:
Is this not just the list index in `day_names`?
```suggestion
        return day_names[inds_in]
```